grakn-core/src/main/java/ai/grakn/util/Schema.java

```diff
@@ -53,6 +53,7 @@ private Schema() {
         RELATES("relates"),
         PLAYS("plays"),
         HAS_SCOPE("has-scope"),
+        //TODO: Remove Casting and Role Player edges
```

Comment:
yes I will do it in this PR. Once https://github.com/graknlabs/grakn/pull/1724 goes in.